backend/azure-functions/src/IsSupervisor/function.json

```diff
@@ -6,7 +6,6 @@
       "direction": "in",
       "name": "req",
       "methods": [
-        "get",
         "post"
```

Comment:
You need here "OPTIONS" besides "POST" (I had issue with that, and I saw that your app also sends OPTIONS request).